.github/workflows/build.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Install sigstore cosign
         uses: sigstore/cosign-installer@1e95c1de343b5b0c23352d6417ee3e48d5bcd422
```

Comment:
can we use `@48866aa521d8bf870604709cd43ec2f602d03ff2` or `@main`?

Reply:
done!